src/sentry/templates/sentry/account/confirm_email/failure.html

```diff
@@ -0,0 +1,15 @@
+{% extends "sentry/bases/auth.html" %}
+
+{% load crispy_forms_tags %}
+{% load i18n %}
+
+{% block title %}{% trans "Confirm Email" %} | {{ block.super }}{% endblock %}
+
+{% block auth_main %}
+    <h2>{% trans "Oops, something happened" %}</h2>
+
+    <p>{% trans "There was an error confirming your email. Please try again or visit your " %}
```

Comment:
@benvinegar @mitsuhiko is it bad to have these `{% trans %}` blocks split like this? Or can we do HTML inside a `{% trans %}` like we do in React?